advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-s3.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>I'm confused by the Hyperparameters section, given the two conflicting tables below. Are there additional hyperparameters I am supposed to set depending on whether I run the Tensorpack or AWS-Samples version? Or are the version-specific params already baked into the images? Can you maybe have Python code here showing the full set of hyperparameters you would use for each image, with some commented out to show what is required vs. optional?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
The ones shown are the common set. One can set add other hyper-parameters to this set as needed. None of these are needed of course if they are same as default values.

Reply:
OK, as long as the defaults are sensible that should be fine. And the Python scripts inside the container are visible to readers if they want to see how to set other optional hyperparameters.